internal/plugin/connectors/http/generic/README.md

```diff
@@ -143,6 +143,26 @@ powerful transformation features.  You can use `printf` for formatting and
 you can compose functions using pipes `|`.  See the text template package docs linked
 above for detailed information on these and other features.
 
+### `oauth1`
+Like `headers` this is another key section. The `oauth1` section is used to generate
+an OAuth1 `Authorization` header. This will override any `Authorization` header that
+you have previously specified in your configuration.
+
+There are four required _keys_ for the `oauth1` section which are as follows:
+1. `consumer_key`
+1. `consumer_secret`
+1. `token`
+1. `token_secret`
```

Comment:
These fields should have descriptions attached here as to what they are (or links to where you can find out more about them)